ReactiveCocoaTests/Swift/PropertySpec.swift

```diff
@@ -73,6 +73,12 @@ class PropertySpec: QuickSpec {
 				mutableProperty = nil
 				expect(signalCompleted).to(beTruthy())
 			}
+
+			it("should not crash in a tuple") {
```

Comment:
This test failed at the start of the PR, then passed with both “fixes,” so it's not a good indicator of whether the crash is gone.

Instead, I've been building https://github.com/Carthage/Carthage/pull/368 against this branch and testing a complex `carthage bootstrap` with it.